frontends/ol-util/src/factories.ts

```diff
@@ -15,14 +15,14 @@ const makePaginatedFactory =
       next?: string | null
       previous?: string | null
     } = {}
-    ): PaginatedResult<T> => {
+    ) => {
       const results = times(pageSize ?? count, () => makeResult())
       return {
         results,
         count,
         next,
         previous
-      }
+      } satisfies PaginatedResult<T>
```

Comment:
Despite the fact that `yarn typecheck` and `yarn build` I was getting a type error in VS Code when *using* `makePaginatedFactory`:

```
The inferred type of 'makeListItemsPaginated' cannot be named without a reference to '../../../node_modules/ol-util/src/interfaces'. This is likely not portable. A type annotation is necessary.ts(2742)
```

This change has no real affect on the signature and made the issue go away.

Others have reported similar problems in https://github.com/microsoft/TypeScript/issues/42873 with no clear resolution.